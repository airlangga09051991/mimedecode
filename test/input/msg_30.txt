From test Sat Feb  1 00:00:00 2014
From: aperson@dom.ain
MIME-Version: 1.0
Content-Type: multipart/digest; boundary=BOUNDARY

--BOUNDARY

Content-Type: text/plain; charset=us-ascii
To: aa@bb.org
From: cc@dd.org
Subject: ee

message 1

--BOUNDARY

Content-Type: text/plain; charset=us-ascii
To: aa@bb.org
From: cc@dd.org
Subject: ee

message 2

--BOUNDARY--
