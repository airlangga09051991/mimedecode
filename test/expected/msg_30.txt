From test Sat Feb  1 00:00:00 2014
From: aperson@dom.ain
MIME-Version: 1.0
Content-Type: multipart/digest; boundary=BOUNDARY

--BOUNDARY

To: aa@bb.org
From: cc@dd.org
Subject: ee
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
X-MIME-Autoconverted: from us-ascii to utf-8 by test id mimedecode

message 1

--BOUNDARY

To: aa@bb.org
From: cc@dd.org
Subject: ee
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
X-MIME-Autoconverted: from us-ascii to utf-8 by test id mimedecode

message 2

--BOUNDARY--
